integrationTests/mock/roundHandlerMock.go

```diff
@@ -20,34 +20,34 @@ func (rndm *RoundHandlerMock) BeforeGenesis() bool {
 }
 
 // Index -
-func (rm *RoundHandlerMock) Index() int64 {
-	return rm.IndexField
+func (rndm *RoundHandlerMock) Index() int64 {
```

Comment:
could have been "mock" instead of "rndm"

Reply:
renamed